.github/scripts/contentful/create.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+if [[ "ON_BRANCH_ENV" == "False" ]]; then
+  echo "No branch Environment required; not deleting environment."
+  exit 0
+fi
+
+if [[ "$CONTENTFUL_ENV_ID" == "Production" ]]; then
+  echo "Target is Production branch; nothing to do."
+  exit 0
+fi
+
+if [[ "$CONTENTFUL_ENV_ID" == "Development" ]]; then
+  echo "Target is Development branch; nothing to do."
+  exit 0
+fi
+
+STATUS_CODE=$(curl --silent \
```

Comment:
why are we not using the [contentful cli](https://www.contentful.com/developers/docs/tutorials/cli/environment-management/)?

Reply:
## Agreed & Consistent Approach

I thought that using the API was an agreed approach as it was specified in [the ticket](https://asaphub.atlassian.net/browse/CRN-1079) which we all refined and estimated, and approved on three previous PRs:

- [Contentful checks](https://github.com/yldio/asap-hub/pull/2175)
- [Contentful teardown](https://github.com/yldio/asap-hub/pull/2192)
- [Contentful teardown hotfix](https://github.com/yldio/asap-hub/pull/2196)

I believe that if we were to switch to CLI we should use it consistently in all of these places. This would also mean checks would be parsing plain-text responses to set mission critical outputs.

## CLI does not provide an advantage

I don't believe that there is any advantage to be gained from using the CLI, it takes a single HTTP call and replaces it with three actions: login, configure, and then whatever the action to be performed is. 

In a long-running interactive session this would make sense because the stored login and configuration would reduce verbosity of subsequent calls, but in our situation we would have to call all three steps every time.

The output is also plain text which is much harder to parse than status codes + JSON.

## CLI just uses API under the hood anyway

The CLI is [just using the HTTP API under the hood anyway](https://github.com/contentful/contentful-cli/blob/master/lib/cmds/login.js), but we'd be making 3x as many HTTP requests in exchange for losing the easily-parsed JSON + status code outputs.